### PR TITLE
 re-enable the test, enable use of gpalloc

### PR DESCRIPTION
**Pull request by MatthewBemis**
passes locally. let's see what jenkins thinks

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [ ] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] Tell your tech lead (TL) that the PR exists if they want to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by aweng98 — APPROVED** [review]
I like review easy PR. send me as many as you have.

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala [hunk 1]
```diff
@@ -73,9 +73,12 @@ class RawlsApiSpec extends FreeSpec with Matchers with CleanUp with BillingFixtu
         val userBWorkspace = Rawls.workspaces.list()(studentBToken)
         userBWorkspace should include(workspaceNameB)
 
+        Orchestration.billing.removeUserFromBillingProject(projectName, studentA.email, Orchestration.billing.BillingProjectRole.User)(ownerAuthToken)
+        Orchestration.billing.removeUserFromBillingProject(projectName, studentB.email, Orchestration.billing.BillingProjectRole.User)(ownerAuthToken)
```
Comment (helgridly) [inline automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala]: not for this PR, but the fact that this was missing before makes me wonder if `addUserToBillingProject` should register its own cleanup

**Review by helgridly — APPROVED** [review]

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala (hunk 1 above)
Comment (aweng98) [reply automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala]: good point. `withCleanBillingProject` takes List(emails) parameter that automatically add and remove users from new proj.

Comment (MatthewBemis) [reply automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala]: @helgridly i had it registering cleanups as a first iteration but the timing of things was strange. the cleanUps execute after the gpalloc'd project is released so it's not able to actually remove the users. i'll note a TODO with this information

@aweng98 i'd use that but it automatically adds them as project Owners which i'm not sure is appropriate for this test

Comment (helgridly) [reply automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala]: oh blergh. there might be potential to attach cleanup to the project-object that joel made recently, but this definitely Requires Some Thought

Comment (breilly2) [reply automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala]: @MatthewBemis You need to insert a `withCleanUp` in the fixtures to control when the registered clean-ups are run. I can help with that.

**Pull request merged**